import_data_into_db.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from sqlalchemy import Column, Float, Integer, String, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+Base = declarative_base()
+
+
+class Animes(Base):
+    __tablename__ = "animes"
+
+    anime_id = Column("anime_id", Integer, primary_key=True)
+
+    name = Column("name", String)
+
```

Comment:
No need to put extra empty lines between each field.
That's making the reading more difficult.

Reply:
The empty lines deleted in commit # e99fdef